niflow_manager/data/templates/base/.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+jobs:
+  build:
+    machine:
+      image: circleci/classic:201710-02
+    working_directory: /home/circleci/nfm_tests
+    steps:
+      - checkout:
+          path: /home/circleci/nfm_tests
+      - run:
+          name: Change to Python 3.7+ and install python dependencies
+          command: |
+            curl -fsSLO https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh -b -p ~/conda
+            rm Miniconda3-latest-Linux-x86_64.sh
+            echo "export PATH=~/conda/bin:$PATH" >> ~/.bashrc
+            source ~/.bashrc
+            conda config --system --prepend channels conda-forge
+            conda update -n base -yq conda
+            conda update -yq --all
+            pip install git+https://github.com/djarecka/niflow-manager.git@new_testkraken
+            #pip install --no-cache-dir -e /home/circleci/nfm_tests[dev]
```

Comment:
Can we do this more simply? As this is a template for users, as simple a configuration as possible would be ideal. Along with comments to explain how to expand it to do more interesting things and URLs to useful resources.

I can make a PR against this, if you'd like.

Reply:
sure, please create a PR if you have something specific in mind

Reply:
Let's merge for now, and I'll make a PR against master as I find time...